### PR TITLE
App: Next.js

### PR DESCRIPTION
**Pull request by donskov**
Rewrite library [website](https://gammacv.com/) (https://github.com/PeculiarVentures/GammaCV/tree/master/app) using [next.js](https://nextjs.org/) for better search engine.

### CODE REVIEW

**Comment by donskov** [comment]
NPM audit report:

```
$ npm audit
                                                                                
                       === npm audit security report ===                        
                                                                                
found 0 vulnerabilities
in 817 scanned packages
```

Snyk report message:
```
Affected module: glob-parent@5.1.1
Introduced through:   next@10.1.3  ›  chokidar@3.5.1  ›  glob-parent@5.1.1
```

`chokidar` repo issue:
https://github.com/paulmillr/chokidar/issues/1076

**Comment by donskov** [comment]
@apilguk @WorldThirteen Please review it when you have a time.

File: app/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "compilerOptions": {
+    "target": "es5",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": false,
```
Comment (WorldThirteen) [inline app/tsconfig.json:11]: Do we have a meaningful reason to disable strict mode?

File: app/src/pages/example/params.tsx [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+        </Box>
+      );
+    }
+
+    return null;
+  }
+}
+
+// @ts-ignore
+ParamsWrapper.contextTypes = {
```
Comment (WorldThirteen) [inline app/src/pages/example/params.tsx:210]: use a static property declaration instead?

File: app/src/pages/example/index.tsx [hunk 3]
```diff
@@ -0,0 +1,689 @@
...
+
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
+// @ts-ignore
+ExamplePage.contextTypes = {
```
Comment (WorldThirteen) [inline app/src/pages/example/index.tsx:685]: static type declaration instead?

File: app/src/pages/example/index.tsx [hunk 4]
```diff
@@ -0,0 +1,689 @@
...
+
+    if (destroy && isPlaying) {
+      this.sess.destroy();
+    }
+    window.cancelAnimationFrame(this.timeoutRequestAnimation);
+
+    this.setState({ isPlaying: false });
+  };
+
+  checkRerender = (arr: any) => {
```
Comment (WorldThirteen) [inline app/src/pages/example/index.tsx:351]:
```suggestion
  checkRerender = (arr: Uint8Array) => {
```

File: app/src/pages/docs/index.tsx [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+    <main
+      className={s.main}
+    >
+      <div className={s.m_width}>
+        <div className={clx(s.doc, 'b1', 'text_black')}>
+          <ReactMarkdown
+            allowDangerousHtml
+            children={data}
+            plugins={[gfm, slug]}
+            renderers={{
```
Comment (WorldThirteen) [inline app/src/pages/docs/index.tsx:31]: nitpick: move renders fns out of jsx?

File: app/pages/_document.tsx [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+      <script async src={`https://www.googletagmanager.com/gtag/js?id=${process.env.GOOGLE_ANALYTICS}`} />
+      <script
+        dangerouslySetInnerHTML={{
+          __html: `
+            window.dataLayer = window.dataLayer || [];
+            function gtag() {
+              dataLayer.push(arguments);
+            }
+            gtag('js', new Date());
+            gtag('config', ${process.env.GOOGLE_ANALYTICS});
```
Comment (WorldThirteen) [inline app/pages/_document.tsx:21]: I've specify GOOGLE_ANALYTICS=helloworld and got:
image

It seems `""` are missing.

File: app/pages/_document.tsx [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+}
+
+function Rollbar() {
+  return (
+    <>
+      <script
+        dangerouslySetInnerHTML={{
+          __html: `
+            var _rollbarConfig = {
+              accessToken: ${process.env.ROLLBAR_API_KEY},
```
Comment (WorldThirteen) [inline app/pages/_document.tsx:36]: I've specified ROLLBAR_API_KEY=awikey and got:
image

It seems "" are missing.

**Review by WorldThirteen — CHANGES_REQUESTED** [review]

**Review by WorldThirteen — APPROVED** [review]
SGTM.

**Pull request merged**